packages/hd-transport/src/types/messages.ts

```
@@ -133,6 +139,25 @@ export type AptosMessageSignature = {
   address: string;
 };
 
+// BenchmarkListNames
+export type BenchmarkListNames = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)